pkg/es/config/esrollovercfg.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
```

Comment:
The file can be called simply `rollover.go` - both `es` and `cfg` are implied by the package.